spinalcordtoolbox/reports/qc2.py

```diff
@@ -280,7 +280,7 @@ def sct_deepseg(
         elif "totalspineseg" in argv:
             sct_deepseg_spinal_rootlets_t2w(
                 imgs_to_generate, fname_input, fname_seg, fname_seg2, species,
-                radius=(40, 40), outline=False)
```

Comment:
Testing `outline=True` with `totalspineseg`, the QC now looks like this:

![image](https://github.com/user-attachments/assets/1b21e53b-e280-43cd-b793-5ea9225f1430)

Splendid!!

Reply:
Woot! Thanks for testing this! I'll mark the PR as non-draft, then.